.github/workflows/cd-pipeline.yaml

```diff
@@ -112,50 +111,91 @@ jobs:
           VIB_ENV_S3_URL: s3://${{ secrets.AWS_S3_BUCKET }}/bitnami
           VIB_ENV_S3_USERNAME: ${{ secrets.AWS_ACCESS_KEY_ID }}
           VIB_ENV_S3_PASSWORD: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+  update-index:
+    runs-on: ubuntu-latest
+    needs:
+      - vib-publish
+    name: Update charts index
+    outputs:
+      result: ${{ steps.update-index.outputs.result }}
+    if: ${{ needs.vib-publish.result == 'success' }}
+    steps:
+      - uses: actions/download-artifact@v3
+        with:
+          path: ~/artifacts
       # If we perform a checkout of the master branch, we will find conflicts with the submodules
       - uses: actions/checkout@v2
         with:
           ref: 'index'
           path: index
-          token: ${{ secrets.BITNAMI_BOT_TOKEN }}
```

Comment:
Thinking about this now, I'm not sure we can remove this. As I see it, we need write access to push anything to master in `bitnami/charts` and an unauthenticated user won't be able to do it. It's strange we are using this here for no reason.

Reply:
Hmm I will keep it just in case (and maybe because it is more resilient for the future) but in my own tests, removing the token != creating commits as an unauthenticated user. This can be related to the fact that we are only enforcing the signed-off (DCO) and not a real GPG-sign:

<img width="1541" alt="Screenshot 2022-07-25 at 16 39 52" src="https://user-images.githubusercontent.com/24253844/180803857-e8d6295e-c871-4d62-8702-e1b49afcbdf4.png">